cms/djangoapps/contentstore/views/course.py

```diff
@@ -1655,19 +1655,15 @@ def _get_course_creator_status(user):
 def _get_library_creator_status(user):
     """
     Helper method for returning the library creation status for a particular user,
-    taking into account the values of DISABLE_LIBRARY_CREATION and LIBRARIES_ENABLED.
+    taking into account the value LIBRARIES_ENABLED.
 
     """
 
     if not LIBRARIES_ENABLED:
-        library_creator_status = 'disallowed_for_this_site'
+        return False
     elif user.is_staff:
-        library_creator_status = 'granted'
-    elif CourseCreatorRole().has_user(user):
-        library_creator_status = 'granted'
-    elif settings.FEATURES.get('DISABLE_LIBRARY_CREATION', False):
-        library_creator_status = 'disallowed_for_this_site'
+        return True
+    elif settings.FEATURES.get('ENABLE_CREATOR_GROUP', False):
+        return CourseCreatorRole().has_user(user)
     else:
-        library_creator_status = 'disallowed_for_this_site'
-
-    return library_creator_status
+        return False
```

Comment:
Shouldn't this default be `True`?  If `ENABLE_CREATOR_GROUP` is False, that means every user is allowed to create courses and libraries. So this should return True. Otherwise, there's no way to allow non-staff users to create libraries.

(And note that the global "staff" is a pretty powerful permission and is not given to many users; it doesn't mean "instructors" or anything like that).